### PR TITLE
Clean up fixtures and codebase hardening

### PR DESCRIPTION
**Pull request by zilchms**

### CODE REVIEW

File: manifests/firewall.pp [hunk 1]
```diff
@@ -11,9 +11,9 @@
   if $monit::httpd_bool and $monit::manage_firewall_bool {
     if defined('::firewall') {
       firewall { "${monit::httpd_port} allow Monit inbound traffic":
-        action => 'accept',
-        dport  => $monit::httpd_port,
-        proto  => 'tcp',
+        jump  => 'accept',
```
Comment (jay7x) [inline manifests/firewall.pp:14]: I guess it was renamed in the firewall module some time ago, right?

File: manifests/init.pp [hunk 2]
```diff
@@ -132,58 +132,51 @@
   String                                  $service_name              = $monit::params::service_name,
   Optional[Integer[1]]                    $start_delay               = $monit::params::start_delay,
 ) inherits monit::params {
-  # <stringified variable handling>
-  if is_string($httpd) == true {
+  if $httpd =~ String {
     $httpd_bool = str2bool($httpd)
   } else {
     $httpd_bool = $httpd
   }
```
Comment (jay7x) [inline manifests/init.pp:139]: Maybe it's better to change to
```puppet
  $httpd_bool = Boolean($httpd)
```

File: manifests/firewall.pp (hunk 1 above)
Comment (jay7x) [reply manifests/firewall.pp:14]: I see the answer in the commit history, nvm

Comment (zilchms) [reply manifests/firewall.pp:14]: I have been searching for the Release of puppetlabs-firewall, that introduces the jump and drops the action parameter. I would have expected that this change would brake the CI somehow, but I doesnt. I will see if I find a good solution to this whole thing

File: manifests/init.pp (hunk 2 above)
Comment (zilchms) [reply manifests/init.pp]: I decided to do this via codebase hardening of the parameter types

**Review by rwaffen — APPROVED** [review]

**Pull request merged**